tools/template.xml.in

```diff
@@ -1,113 +1,120 @@
+      <address type="drive" controller="0" bus="0" target="0" unit="0" />
     </disk>
-    <disk type='file' device='disk'>
-      <driver name='qemu' type='raw'/>
-      <source file='VMDIR/BaseSystem.img'/>
-      <target dev='sdb' bus='sata'/>
-      <address type='drive' controller='0' bus='0' target='0' unit='1'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="qcow2" cache="writeback" io="threads" />
+      <source file="BOXESHOME/macOS.qcow2" />
+      <target dev="sdb" bus="sata" />
+      <address type="drive" controller="0" bus="0" target="0" unit="1" />
     </disk>
-    <controller type='usb' index='0' model='ich9-ehci1'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1d' function='0x7'/>
+    <disk type="file" device="disk">
+      <driver name="qemu" type="raw" />
+      <source file="BOXESHOME/BaseSystem.img" />
+      <target dev="sdc" bus="sata" />
+      <address type="drive" controller="0" bus="0" target="0" unit="2" />
+    </disk>
+    <controller type="usb" index="0" model="ich9-ehci1">
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1d" function="0x7" />
     </controller>
-    <controller type='usb' index='0' model='ich9-uhci1'>
-      <master startport='0'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1d' function='0x0' multifunction='on'/>
+    <controller type="usb" index="0" model="ich9-uhci1">
+      <master startport="0" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1d" function="0x0" multifunction="on" />
     </controller>
-    <controller type='usb' index='0' model='ich9-uhci2'>
-      <master startport='2'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1d' function='0x1'/>
+    <controller type="usb" index="0" model="ich9-uhci2">
+      <master startport="2" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1d" function="0x1" />
     </controller>
-    <controller type='usb' index='0' model='ich9-uhci3'>
-      <master startport='4'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1d' function='0x2'/>
+    <controller type="usb" index="0" model="ich9-uhci3">
+      <master startport="4" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1d" function="0x2" />
     </controller>
-    <controller type='sata' index='0'>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x1f' function='0x2'/>
+    <controller type="sata" index="0">
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x1f" function="0x2" />
     </controller>
-    <interface type='network'>
-      <mac address='52:54:00:92:d4:7b'/>
-      <source network='default'/>
-      <model type='e1000-82545em'/>
-      <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x0'/>
+    <interface type="user">
+      <mac address="52:54:00:92:d4:7b" />
+      <model type="e1000-82545em" />
+      <address type="pci" domain="0x0000" bus="0x00" slot="0x02" function="0x0" />
```

Comment:
@kowalski7cc I've recently learned about network interfaces when running a QEMU VM as unprivileged user.

The `usermode` network interface is fine for the VM to access the external internet, but unsuitable if we want to access the VM from the host. My understanding is that if we remove the bridge, users will not be able to network access the VM by any means (unless fiddling with iptables or other port-forwarding hacks).

Configuring the bridge to be accessible from a `qemu:///session` VM is out of scope here, but imo it could be a good idea to mention this somewhere.